tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -3,7 +3,20 @@
 AS part of verification the script  perform the following tasks-
```

Comment:
The code changes with regard to selecting the value for `osd_scrub_auto_repair_num_errors` based on number of Inconsistent objects could be implemented here as well.

Reply:
I Provide the details in the below steps.